.github/workflows/test-carthage-integration.yml

```diff
@@ -12,27 +12,28 @@ on:
 jobs:
 
   carthage:
-    runs-on: macos-12
+    runs-on: macos-latest
```

Comment:
The combination of macos-latest + a hardcoded xcode version might break in the future - In general I like the idea of using the latest macos but it might result in random workflow failures - maybe better to choose a proven macos version 🤔